ams/server/src/main/java/com/netease/arctic/server/terminal/TerminalManager.java

```diff
@@ -247,7 +249,9 @@ private String catalogConnectorType(CatalogMeta catalogMeta) {
       }
     } else if (catalogType.equalsIgnoreCase(CatalogType.HIVE.name())
         || catalogType.equalsIgnoreCase(CatalogType.HADOOP.name())) {
-      if (StringUtils.containsIgnoreCase(tableFormats, TableFormat.MIXED_HIVE.name())
+      if (tableFormatSet.size() > 1) {
+        return "unified";
+      } else if (tableFormatSet.contains(TableFormat.MIXED_HIVE)
           || StringUtils.containsIgnoreCase(tableFormats, TableFormat.MIXED_ICEBERG.name())) {
         return "arctic";
       } else if (StringUtils.containsIgnoreCase(tableFormats, TableFormat.ICEBERG.name())) {
```

Comment:
Maybe you can refactor all codes like `StringUtils.contains(tableFormats)` to `tableFormat.contains( ... )`